build/kubefile/parser/kubefile.go

```diff
@@ -179,6 +192,18 @@ func (kp *KubefileParser) generateResult(mainNode *Node) (*KubefileResult, error
 		}
 	}
 
+	// register app with all env list.
+	for appName, appEnv := range result.AppEnvMap {
+		app := result.Applications[appName]
```

Comment:
maybe `result.Applications[appName].AppEnv = appEnv` is better

Reply:
this is not a copy, `result.Applications` is a interface, and i am trying to register v1app here with all env list.